.gitmodules

```
@@ -10,3 +10,7 @@
 	path = micro-ros-stm32_embeddedrtps/micro-ros_embeddedrtps/micro_ros_stm32cubemx_utils
 	url = https://github.com/mROS-base/micro_ros_stm32cubemx_utils
 	branch = galactic
+[submodule "mros2-posix"]
+	path = mros2-posix
+	url = git@github.com:mROS-base/mros2-posix
```

```suggestion
	url = https://github.com/mROS-base/mros2-posix
```

Of course I understand ssh is better than https. But for those unfamiliar with GitHub, this notation can cause trouble (in fact, I have dealt with people who have trouble with this several times :D